### PR TITLE
add ext-imagick

### PR DESCRIPTION
**Pull request by mzeromskizl**

### CODE REVIEW

File: wercker-box.yml [hunk 1]
```diff
@@ -23,3 +23,4 @@ script: |
     rm -r memcached-${pecl_memcached_ver}
     echo "extension=memcached.so" > 20-memcached.ini
     phpenv config-add 20-memcached.ini
+    echo | pecl install imagick
```
Comment (prgTW) [inline wercker-box.yml]: what's the `echo |` for?

Comment (mzeromskizl) [inline wercker-box.yml]: i dont know - copied from line 12,13

Comment (mlebkowski) [inline wercker-box.yml]: it doesn’t sent stdin to the command

**Comment by mlebkowski** [comment]
@mzeromskizl, unless something has changed at wercker, you have to bump the version number in order to publish an update to a box. see 5a4503a3cf019d53e664705dd2c441ab7d1a16b8 for example

**Pull request merged**